### ISSUE TITLE
 onSlidingComplete ignored when onValueChange set

### ISSUE THREAD

**Issue by cyril36** [Sharcoux/slider#89]
**Describe the bug**
If i set both onValueChange and onSlidingComplete, onSlidingComplete does not seems to be executed when i release the button while i think it should be the latest event that would occur on top of onValueChange
here is an example :
```
<Slider
   ...
   onValueChange={(value)=>setAreaSize(value)}
   onSlidingComplete={(value) => {
              setAreaSize(0)
            }}
/>
```

**To Reproduce**
Here is the code to reproduce it https://snack.expo.dev/@cyril.moreau/sharcoux-slider-demo
**Expected behavior**
When the press is released the number should be 0 

**Please complete the following information:**
 - Android
 - Browser  chrome,
 - @react-native-assets/slider "^7.0.5"

**Comment by Sharcoux** [comment]
Ok, the problem is not that onSlidingComplete is not being executed. It's just that an extra onValueChange is being executed. I'm gonna fix that.

**Comment by Sharcoux** [comment]
@cyril36 fixed in version 7.0.7 along with a problem that caused multiple onChange events to be fired for the same value.

**Comment by cyril36** [comment]
I have tested it and it looks great!
thank you for your fix
